torchtext/nn/modules/multiheadattention.py

```diff
@@ -200,13 +200,13 @@ def forward(self, query: torch.Tensor, key: torch.Tensor, value: torch.Tensor,
 
 class InProjContainer(torch.nn.Module):
     def __init__(self, query_proj, key_proj, value_proj):
-        r"""A in-proj container to process inputs.
+        r"""A in-proj container to project query/key/value in MultiheadAttention. This module happens before reshaping
```

Comment:
A picture could really be useful here or at least a reference to a picture in a paper.